bin/rise

```diff
@@ -41,8 +41,12 @@ if Rise::Util.first_run?
 end
 Rise::Util.check_for_update!
 result_url = ''
-uploader = Rise::Transport::Uploader.new(options[:directory]) unless options[:directory].nil?
-uploader = Rise::Transport::Uploader.new(Dir.pwd) if options[:directory].nil?
+uploader = Rise::Transport::Uploader.new(options[:directory] || Dir.pwd)
+
+if uploader.total_files_size > 52428800
+  puts Paint["Max file size reached (#{uploader.total_files_size} > 52428800)", '#FF0000']
```

Comment:
Maybe change `52428800` to `50MB`

Reply:
Fixed @cbrnrd

Reply:
great, thanks. I’ll test this as soon as possible and merge it if it works well

Reply:
With your most recent changes, im getting: 
```
bin/rise:47:in `<main>': undefined method `megabytes' for 50:Integer (NoMethodError)
```

I think keep the original `if uploader.total_files_size > 52428800` but change the `52428800` in the puts statement to 50MB